rapid7_vulndb/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM komand/python-3-plugin:2
+FROM komand/python-3-37-plugin:3
```

```suggestion
FROM komand/python-3-37-slim-plugin:3
```
Use slim if possible